typings/index.d.ts

```diff
@@ -1,31 +1,44 @@
-import {getQueriesForElement} from 'dom-testing-library'
+import {queries, BoundFunction} from 'dom-testing-library'
 
 export * from 'dom-testing-library'
 
-type GetsAndQueries = ReturnType<typeof getQueriesForElement>
+interface Query extends Function {
+  (container: HTMLElement, ...args): HTMLElement[] | HTMLElement | null
```

Comment:
@chochihim @kentcdodds heads up, the `...args` expression here causes TS builds to fail when `--noImplicitAny` is set.

```
Type error: Rest parameter 'args' implicitly has an 'any[]' type.  TS7019

    4 |
    5 | interface Query extends Function {
  > 6 |   (container: HTMLElement, ...args): HTMLElement[] | HTMLElement | null
      |                            ^
    7 | }
    8 |
    9 | interface Queries {
```

Reply:
😬

Could you please open a PR to fix this @joeporpeglia.

Reply:
Gotcha! #260